### PR TITLE
chore/TEN-54-Node-18-testing

### PR DESCRIPTION
**Pull request by JancoHY**

### CODE REVIEW

File: packages/happn-3/test/__fixtures/test/temp_test_lookup_db [hunk 1]
```diff
@@ -0,0 +1 @@
...":[]},\"modified\":{\"name\":\"modified\",\"dirty\":false,\"values\":[]}},\"constraints\":null,\"uniqueNames\":[\"path\"],\"transforms\":{},\"objType\":\"happn\",\"dirty\":true,\"cachedIndex\":null,\"cachedBinaryIndex\":null,\"cachedData\":null,\"adaptiveBinaryIndices\":true,\"transactional\":false,\"cloneObjects\":false,\"cloneMethod\":\"parse-stringify\",\"asyncListeners\":false,\"disableMeta\":false,\"disableChangesApi\":true,\"disableDeltaChangesApi\":true,\"autoupdate\":false,\"serializableIndices\":true,\"disableFreeze\":true,\"ttl\":null,\"maxId\":0,\"DynamicViews\":[],\"events\":{\"insert\":[],\"update\":[],\"pre-insert\":[],\"pre-update\":[],\"close\":[],\"flushbuffer\":[],\"error\":[],\"delete\":[null],\"warning\":[null]},\"changes\":[],\"dirtyIds\":[]}],\"databaseVersion\":1.5,\"engineVersion\":1.5,\"autosave\":false,\"autosaveInterval\":5000,\"autosaveHandle\":null,\"throttledSaves\":true,\"options\":{\"serializationMethod\":\"normal\",\"destructureDelimiter\":\"$<\\n\"},\"persistenceMethod\":\"fs\",\"persistenceAdapter\":null,\"verbose\":false,\"events\":{\"init\":[null],\"loaded\":[],\"flushChanges\":[],\"close\":[],\"changes\":[],\"warning\":[]},\"ENV\":\"NODEJS\"}"}
```
Comment (southbite) [inline packages/happn-3/test/__fixtures/test/temp_test_lookup_db:1]: We need to investigate why this is still landing up there - do a grep for test_lookup_db, maybe the test unlinks it, but not the temp file

File: packages/happn-db-provider-mongo/test/__fixtures/data/functional_mixed.nedb [hunk 2]
```diff
@@ -1,13 +1,13 @@
...":[]},\"modified\":{\"name\":\"modified\",\"dirty\":false,\"values\":[]}},\"constraints\":null,\"uniqueNames\":[\"path\"],\"transforms\":{},\"objType\":\"happn\",\"dirty\":true,\"cachedIndex\":null,\"cachedBinaryIndex\":null,\"cachedData\":null,\"adaptiveBinaryIndices\":true,\"transactional\":false,\"cloneObjects\":false,\"cloneMethod\":\"parse-stringify\",\"asyncListeners\":false,\"disableMeta\":false,\"disableChangesApi\":true,\"disableDeltaChangesApi\":true,\"autoupdate\":false,\"serializableIndices\":true,\"disableFreeze\":true,\"ttl\":null,\"maxId\":0,\"DynamicViews\":[],\"events\":{\"insert\":[],\"update\":[],\"pre-insert\":[],\"pre-update\":[],\"close\":[],\"flushbuffer\":[],\"error\":[],\"delete\":[null],\"warning\":[null]},\"changes\":[],\"dirtyIds\":[]}],\"databaseVersion\":1.5,\"engineVersion\":1.5,\"autosave\":false,\"autosaveInterval\":5000,\"autosaveHandle\":null,\"throttledSaves\":true,\"options\":{\"serializationMethod\":\"normal\",\"destructureDelimiter\":\"$<\\n\"},\"persistenceMethod\":\"fs\",\"persistenceAdapter\":null,\"verbose\":false,\"events\":{\"init\":[null],\"loaded\":[],\"flushChanges\":[],\"close\":[],\"changes\":[],\"warning\":[]},\"ENV\":\"NODEJS\"}"}
```
Comment (southbite) [inline packages/happn-db-provider-mongo/test/__fixtures/data/functional_mixed.nedb:1]: check why nedb extension is used here - this is a loki db, find out whether we want this in code base

File: packages/happn-util-crypto/package.json [hunk 3]
```diff
@@ -8,7 +8,7 @@
   },
   "main": "lib/crypto",
   "scripts": {
-    "test": "_mocha --recursive --exit silence.js test/*",
```
Comment (southbite) [inline packages/happn-util-crypto/package.json:11]: add tests to github workflows

File: packages/happner-2/test/integration/web/component-web.js [hunk 4]
```diff
@@ -47,7 +47,7 @@ require('../../__fixtures/utils/test_helper').describe({ timeout: 15e3 }, (test)
         },
         web: {
           routes: {
-            // http://localhost:3001/neptronicUI/...
```
Comment (southbite) [inline packages/happner-2/test/integration/web/component-web.js:50]: take out this dead code

File: packages/happner-client/test/example.com.cert [hunk 5]
```diff
@@ -1,17 +1,17 @@
 -----BEGIN CERTIFICATE-----
-MIICpDCCAYwCCQDc/3wrjZJF+DANBgkqhkiG9w0BAQsFADAUMRIwEAYDVQQDDAls
```
Comment (southbite) [inline packages/happner-client/test/example.com.cert:2]: figure out why these get checked in every time, they may need to be git deleted and ignored

**Review by southbite — CHANGES_REQUESTED** [review]

File: packages/happner-client/test/example.com.cert (hunk 5 above)
Comment (JancoHY) [reply packages/happner-client/test/example.com.cert:2]: See change to happn-3 transport service above. This should no longer be recreated on each test run

File: packages/happn-db-provider-mongo/test/__fixtures/data/functional_mixed.nedb (hunk 2 above)
Comment (JancoHY) [reply packages/happn-db-provider-mongo/test/__fixtures/data/functional_mixed.nedb]: Looking at the test I believe it should be a Loki DB. Renamed accordingly, and made db-provider explicit in test

File: packages/happner-client/test/example.com.cert (hunk 5 above)
Comment (JancoHY) [reply packages/happner-client/test/example.com.cert:2]: See changes to happn-3 transport service...

**Review by southbite — APPROVED** [review]

**Pull request closed without merging**